src/libraries/Common/src/Interop/Windows/WinSock/Interop.WSARecv.cs

```diff
@@ -31,7 +31,7 @@ internal static partial class Winsock
             IntPtr completionRoutine)
         {
             Debug.Assert(!buffers.IsEmpty);
-            fixed (WSABuffer* buffersPtr = &MemoryMarshal.GetReference(buffers))
+            fixed (WSABuffer* buffersPtr = buffers)
```

Comment:
Is this change necessary for correctness?

Reply:
Maaaybe? Will use `buffers.GetPinnableReference()` which will return `null` when `.Length == 0` rather than an overwriable addresses; though otherwise could be seen as churn...

Reply:
Changed back